src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1261,6 +1270,49 @@ def _get_log(url, user_name, password, log_file=None):
     r.release_conn()
 
 
+def _print_site_logs(api_string, number_of_lines, byte_limit, http, headers):
+    import sys
+    href_count = api_string.count('https')
+
+    std_encoding = sys.stdout.encoding
+    if href_count > 1:
+        logger.info('Will show last %i lines of logs per instance', number_of_lines)
+
+    for x in range(0, href_count):
+        index_of_path = api_string.index('\",\"path')
+        href_str = api_string[api_string.index("https"):index_of_path]
+        size = api_string[api_string.index('\"size\":') +
+                          len('\"size\":'):api_string.index(',\"href\"')]
+        logger.warning('\nSite Instance #%i', x + 1)
+        logger.info('\nUsing endpoint: %s', href_str)
+        url = href_str
+        byte_limit_exceeded = (int(size) > byte_limit)
+        if byte_limit_exceeded is True:
+            headers.update({'Range': 'bytes=-512000'})  # only get the last 512k
+
+        r = http.request(
+            'GET',
+            url,
+            headers=headers,
+            preload_content=False
+        )
+        lines = (r.data.decode(encoding='utf-8', errors='replace')
+                 .encode(std_encoding, errors='replace')
+                 .decode(std_encoding, errors='replace')
+                 .replace('\\n', '\n'))
+        lines_split = lines.splitlines()
+        lines_split_length = len(lines_split)
+        if byte_limit_exceeded is True:
+            if lines_split_length < int(number_of_lines):
+                logger.warning("Hit internal limit of %s bytes."
+                               "Please download to see full logs", byte_limit)
+        if int(number_of_lines) > lines_split_length:
+            number_of_lines = lines_split_length
+        last_n_lines = ('\n'.join(lines_split[-int(number_of_lines)::])).lstrip('\n')
```

Comment:
will `lines_split[:number_of_lines])` be more readable?

Reply:
This will give me the first 'n' lines. I want the last 'n' lines.